src/shared/building.js

```diff
@@ -15,19 +15,28 @@ export default class Building {
     this.col = col;
     this.team = team;
     this.size = size;
-    this.health = health;
     this.powered = false;
     this.isSelected = false;
+    this.health = this.team === NEUTRAL ? 1 : health;
+    this.delta = 0;
```

Comment:
Delta time probably shouldn't be stored as a member variable, and doesn't seem to be used anywhere.